### PR TITLE
Kartik khode

### PR DESCRIPTION
**Pull request by Kartiklk**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2499"></a>

### CODE REVIEW

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

File: data/kartiklk.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    "name": "Kartik khode",
+    "displayStatsPublic": true,
+    "type": "personal",
+    "bio": "Backend Developer | Open Source and Tech Enthusiast",
+    "avatar": "https://github.com/kartiklk.png",
+    "tags": ["NodeJS", "ExpressJS", "MongoDB", "Javascript", "OpenSource"],
+    "socials": [
+      { "icon": "FaTwitter", "url": "https://twitter.com/kartikkhode21" },
+      { "icon": "FaGithub", "url": "https://github.com/Kartiklk" },
+      { "icon": "FaLinkedin", "url": "https://www.linkedin.com/in/kartik-khode-32a7a325a/" }
```
Comment (eddiejaoude) [inline data/kartiklk.json:11]: I think you are missing a command (`,`) at the end of this line

```suggestion
      { "icon": "FaLinkedin", "url": "https://www.linkedin.com/in/kartik-khode-32a7a325a/" },
```

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Missing comma

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Pull request merged**